pkg/kubelet/winstats/perfcounter_nodestats.go

```diff
@@ -168,25 +168,25 @@ func (p *perfCounterNodeStatsClient) collectMetricsData(cpuCounter, memWorkingSe
 	cpuValue, err := cpuCounter.getData()
 	cpuCores := runtime.NumCPU()
 	if err != nil {
-		klog.Errorf("Unable to get cpu perf counter data; err: %v", err)
+		klog.ErrorS(err, "Unable to get cpu perf counter data")
 		return
 	}
 
 	memWorkingSetValue, err := memWorkingSetCounter.getData()
 	if err != nil {
-		klog.Errorf("Unable to get memWorkingSet perf counter data; err: %v", err)
+		klog.ErrorS(err,"Unable to get memWorkingSet perf counter data")
```

Comment:
```suggestion
		klog.ErrorS(err, "Unable to get memWorkingSet perf counter data")
```

gofmt is failing on this line.

Reply:
Thanks @ehashman. Changed as suggested.